internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json

```diff
@@ -3,33 +3,6 @@
     "geoId/05": {
       "statVars": [
         "AmountFarmInventory_WinterWheatForGrain",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSAccommodationFoodServices_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSAdministrativeSupportWasteManagementRemediationServices_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSAgricultureForestryFishingHunting_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSArtsEntertainmentRecreationAccommodationFoodServices_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSArtsEntertainmentRecreation_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSConstruction_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSDurableGoodsManufacturing_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSEducationalServicesHealthCareSocialAssistance_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSEducationalServices_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSFinanceInsuranceRealEstateRentalLeasing_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSFinanceInsurance_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSHealthCareSocialAssistance_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSInformation_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSManagementOfCompaniesEnterprises_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSManufacturing_RealValue",
-        "Amount_EconomicActivity_GrossDomesticProduction_NAICSMiningQuarryingOilGasExtraction_RealValue",
```

Comment:
@manishvats2 @krishnaswamypradeep Can you please confirm if these SVs from BEAGDPv2 are expected to be deleted in the latest version in autorefresh. They were present in the manual refresh with cl/770950461.